common/config/rush/pnpm-lock.yaml

```diff
@@ -1378,7 +1378,7 @@ packages:
   /@types/body-parser/1.19.0:
     dependencies:
       '@types/connect': 3.4.34
-      '@types/node': 8.10.66
+      '@types/node': 15.12.2
```

Comment:
The `@types/node` version keeps changing based on whose machine the rush update PRs usually come from. Anybody know the mystery here? Or is ti just dependent on the version of node on the PR author's machine? 

cc @xirzec, @chradek

Reply:
Hmm, it should not be dependent on the version of node on the author's machine because that doesn't have any influence on the types.

My pet theory was that the author may have inadvertently pulled in a new version while working in the repo across branches, but this is presumably happening now from freshly cloning the repo!

Reply:
I propose we go with this change and then wait and watch what happens next :)

Assuming we will have automated PRs going forward, it should stay the same

Reply:
Seems like some transitive dependencies are pulling it in? I'm fine approving this for now